### DISCUSSION TITLE
Help Wanted

### DISCUSSION THREAD

**Discussion by onrblnt** [realsuayip/django-sozluk#80] [Q&A] [answered]
Selam Şuayip,

Çok güzel iyi hoş ama, bunun kurulumunu yapıp domaine ve hostinge bağlayıp aktif bir websitesi haline getirecek birikimi olmayan benim gibi cahiller ne yapsın? :) Bu aşamalarda ücreti mukabilinde yardım edebilir misin ya da edebilecek birini tanıyor musun? Teşekkürler.

**Comment by realsuayip** [comment, marked as answer]
ben bakamıyorum şu sıralar maalesef. @onrblnt şuradan bir sordurabilirsin:

https://t.me/django_turkey
